partner_financial_risk/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Partner Financial Risk
+======================
+
+Adds a new page in partner to manage its *Financial Risk*.
+
+If any limit is exceed the partner gets forbidden to confirm customer invoice.
```

Comment:
`If any limit is exceeded, you won't be able to confirm any of its invoices unless you are authorized.`